src/web/mjs/connectors/Tapas.mjs

```diff
@@ -47,24 +47,34 @@ export default class Tapas extends Connector {
 
     async _getChapters(manga) {
         let chapterList = [];
+
         let request = new Request(new URL(manga.id, this.url), this.requestOptions);
-        let data = await this.fetchDOM(request, 'div.paging a.paging__button--num');
-        let pageCount = data.length === 0 ? 1 : parseInt(data.pop().href.match(/pageNumber=(\d+)/)[1]);
-        for(let page = 1; page <= pageCount; page++) {
-            let chapters = await this._getChaptersFromPage(manga, page);
-            chapterList.push(...chapters);
+        let data = await this.fetchDOM(request, 'meta[name="twitter:app:url:iphone"]');
+        const series_id = data[0].content.match(/((?:\/series\/)\d+)/)[0];
+
+        let has_next = true;
+        const time = Math.round(new Date().getTime());
+        let page = 1;
+        while (has_next) {
+            request = new Request(new URL(series_id+'/episodes?page='+page+'&sort=OLDEST&init_load=0&since='+time+'&large=true&last_access=0&', this.url), this.requestOptions);
+            request.headers.set('x-referer', this.getRootRelativeOrAbsoluteLink(manga.id, this.url));
+            let response = await this.fetchJSON(request, this.requestOptions);
+            has_next = response.data.pagination.has_next;
+            chapterList.push(...await this._getChaptersFromPage(response.data.body));
+            page++;
         }
+
         return chapterList;
     }
 
-    async _getChaptersFromPage(manga, page) {
-        let request = new Request(new URL(`${manga.id}?pageNumber=${page}`, this.url), this.requestOptions);
-        let data = await this.fetchDOM(request, 'li.content__item  > a');
+    async _getChaptersFromPage(payload) {
```

Comment:
May clarify method name, since it extracts chapters from HTML content instead of fetching it from a page